deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,71 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/*Schrijf hier jouw css*/
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+    background-color: #F2F3D9;
+    font-family: 'Poppins', sans-serif;
+}
+
+nav {
+    display: flex;
+    flex-direction: row;
+    background-color: #151E3F;
+    padding: 40px;
+    justify-content: space-between;
+}
+
+.navUL {
+    display: flex;
+    flex-direction: row;
+    gap: 50px;
+    list-style: none;
+}
+
+a {
+    color: white;
+    text-decoration: none;
+}
+
+.anchor ~ a {
+    justify-content: flex-end;
+}
+
+.header {
+    display: flex;
+    flex-direction: column;
+    padding: 40px;
+}
+
+h1, h3 {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    padding-top: 10px;
+}
+
+.wrapper {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: space-evenly;
+    gap: 40px;
+    padding: 0 40px;
+}
+
+.container {
```

Comment:
Hier mist nog padding, margin en border-radius. 
(dit komt uit de uitwerkingen: 
border-radius: 4px;
    padding: 20px;
    margin: 20px;)

In jouw geval is margin niet nodig volgens mij, padding en border-radius wel.